src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform 300ms;
+
+    &:hover {
+      color: #34568b;
```

Comment:
card title text color has to smoothly change

Reply:
done